analytics/loggable_objects.go

```diff
@@ -0,0 +1,78 @@
+package analytics
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"net/http"
+)
+
+type RequestType string
+
+const (
+	COOKIE_SYNC = "cookie_sync"
+	AUCTION     = "auction"
+	SETUID      = "set_uid"
```

Comment:
Should these three be given a defined type, such as AnalyticsObjectName, to take advantage of type safety?